wasm2c/examples/rot13/Makefile

```diff
@@ -4,9 +4,12 @@ CFLAGS=-I../..
 all: rot13
 
 clean:
-	rm -rf rot13 rot13.wasm rot13.c *.o
+	rm -rf rot13 rot13.wasm rot13.c rot13.h *.o
 
-rot13: main.o rot13.o ../../wasm-rt-impl.o ../../wasm-rt-mem-impl.o -lm
+rot13: main.o rot13.o ../../wasm-rt-impl.o ../../wasm-rt-mem-impl.o
+	$(CC) $(LDFLAGS) -o $@ $^ -lm
+
+main.o: rot13.c
```

Comment:
Would it make more sense to put rot13.h on the right here?  I guess that only makes sense if its easy to also add `rot13.h` to the left hand side of the `rot13.c` rules?

Reply:
I've done that, what do you think?